packages/language-server/test/plugins/typescript/features/SemanticTokensProvider.test.ts

```diff
@@ -73,48 +73,6 @@ describe('TypeScript Plugin#SemanticTokenProvider', () => {
 				type: TokenType.method,
 				modifiers: [TokenModifier.defaultLibrary],
 			},
-			{
```

Comment:
The Semantic Token test tested the Astro global with Astro.props, since this change the token type of that, let's just remove that part from the test

Since this test test other things and there's no reasons semantic tokens would fail on the Astro global in particular, this doesn't really remove coverage